src/process/qgsprocess.cpp

```diff
@@ -261,6 +261,22 @@ int QgsProcessingExec::run( const QStringList &constArgs )
     args.removeAt( verboseIndex );
   }
 
+  int langIndex = args.indexOf (QLatin1String( "--lang"));
+  if ( langIndex >= 0 )
+  {
+    // has already been handled in (qgis_process) main.cpp
+      args.removeAt ( langIndex );  //'--lang'
+      args.removeAt ( langIndex );  //e.g. 'de'
```

Comment:
again, a potential crash here if --lang is used without any corresponding value following it